lib/stdio-wrapper.js

```diff
@@ -0,0 +1,87 @@
+'use strict';
+
+// Adapted from https://github.com/puppeteer/puppeteer/blob/7a2a41f2087b07e8ef1feaf3881bdcc3fd4922ca/src/PipeTransport.js
```

Comment:
if this is objectionable for some reason, this can be rewritten from scratch, it's not too complicated